classes/fields/time.php

```diff
@@ -209,7 +222,10 @@ public function format_time( $options, $js = false ) {
 
 					if ( empty( $format ) ) {
 						$format = pods_v( static::$type . '_format_custom', $options, '' );
-						$format = $this->convert_format( $format, array( 'source' => 'php' ) );
+
+						if ( $js ) {
+							$format = $this->convert_format( $format, array( 'source' => 'php', 'type' => 'time' ) );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 7c8ca97b82e32a04a3046c05e914852d</sub>